### DISCUSSION TITLE
Scale of the design and simulation time?

### DISCUSSION THREAD

**Discussion by 10UAikawa** [NanoComp/meep#2817] [Q&A] [answered]
Hi Meep community this might be a dumb question. I was just wondering does changing the scale of your design to a scale with a smaller volume help simulation time? For example, if I have a Computational cell is 100 x 100 x 275 with a resolution of 0.5, this will take a long time. Could I decrease the size of everything by a factor of 10, since the MEEP units are scale invariant. Then increase the resolution? I have tried this, and it does not really seem to help, in fact it seems to distort the results. I can do this same simulation in HFSS, and it never takes as long. I was just wondering if anybody has any tips to deal with this, or if there are any work arounds?

**Comment by stevengj** [comment, marked as answer]
Rescaling everything by the same factor (i.e. multiply the sizes and times by X and divide resolution and frequency by X) just changes the units, it should not change the results or the runtime at all.

(HFSS is an entirely different type of algorithm, it's FEM frequency domain.  For some problems this is much more efficient than FDTD — especially if you have problems with big variations in lengthscale, or are interested only in narrow bandwidths and/or very dispersive materials.)
